common/assets/javascripts/application.js

```diff
@@ -20,6 +20,7 @@ import MultiFileUpload from '../../components/multi-file-upload/multi-file-uploa
 
 const accessibleAutocomplete = require('accessible-autocomplete')
 const { initAll } = require('govuk-frontend')
+const HcSticky = require('hc-sticky')
```

Comment:
Without being too much of an expert in CSS, could we use [`position: sticky`](https://developer.mozilla.org/en-US/docs/Web/CSS/position) instead of this library? I'm not sure what browsers we support, but it's only IE where this isn't supported.

Reply:
I did consider this, but it was a little more than it staying in the same position and the same form.

Ours now works a little similar to how the GH Pull Requests work in that once you scroll the header out of view, it then appears positioned to the top of the page in a condensed format. I think if we just used `position: sticky` I don't think there would be a way to adjust the styling once it's "stuck".

This library was one of the only ones I could find that also creates the placeholder in the right size so that it doesn't feel too jerky. 

Have a play with it on the [new design layout](https://booksecuremove-pr-1833.herokuapp.com/move/cc721283-e508-4635-ad9b-7f8fc80187b0/beta/warnings) and let me know what you think.

Reply:
Makes sense, it looks good on the preview! 👍🏻